### PR TITLE
Feature/translation to es

### PR DESCRIPTION
**Pull request by scrhall**

### CODE REVIEW

File: locales/es/LC_MESSAGES/help.po [hunk 1]
```diff
@@ -41,30 +46,30 @@ msgstr ""
...
-msgstr ""
+msgstr "Este bot usa un sistema de permisos, por defecto nadie puede modificar tus listas. Para cambiar esto, tienes que ir a la configuración de la lista y pulsar el botón de arriba a la izquierda (👤) y después compartir la lista con quien quieras que pueda editarla. Por favor, recuerda que al marcar la lista como abierta, cualquiera con quien compartas la lista tendrá acceso total a la lista, aunque siempre puedes cerrarla de nuevo :)"
 
 #: helpFuncs.py:38
 msgid ""
 "👤 means, that the list is currently closed and usage is restricted to the owner and users, who already joined the list. Pushing this button will open the list for everyone and will change the icon to 👥.\n"
 "✅⬆ and ✅⬇ will reorder already done tasks to the top/bottom of the list, respectively.\n"
 "↩ to go back to the list."
-msgstr ""
+msgstr "👤 significa que la lista esta cerrada y el uso esta restringido a el dueño y los usuarios que ya se añadieron a la lista, Pulsar esta botón abrirá la lista para cualquiera y cambiara el icono a 👥.\n"
```
Comment (Lunaresk) [inline locales/es/LC_MESSAGES/help.po:68]: May you translate the other two lines of the above strings too, please? :3

File: locales/es/LC_MESSAGES/main.po [hunk 2]
```diff
@@ -17,85 +17,84 @@ msgstr ""
...
+msgstr "Cerrada"
 
 #: do2bot.py:348
 msgid "List access set to {0}"
-msgstr ""
+msgstr "Acceso a la lista establecido a {0}"
 
 #: do2bot.py:348
 msgid "Open"
 msgstr ""
```
Comment (Lunaresk) [inline locales/es/LC_MESSAGES/main.po:93]: Line 92 might need a translation :3 otherwise it will fall back to english

Comment (scrhall) [reply locales/es/LC_MESSAGES/main.po:93]: Done ;)

File: locales/es/LC_MESSAGES/help.po (hunk 1 above)
Comment (scrhall) [reply locales/es/LC_MESSAGES/help.po:68]: wow Sorry ;)

**Review by Lunaresk — APPROVED** [review]

**Pull request merged**